docs/pages/choose-an-edition/teleport-cloud/external-audit-storage.mdx

```diff
@@ -0,0 +1,130 @@
+   your audit log.
+1. A transient storage S3 bucket that temporarily holds results of Athena
+   queries.
+
+## Authentication
+
+Your Teleport Enterprise Cloud cluster will act as an OpenID Connect (OIDC)
+identity provider to authenticate to your organization's AWS account.
+
+The Teleport UI will walk you through the process of configuring the identity
+provider in your account and attaching an IAM role and policy that will allow
+the necessary access to your AWS resources.
+The policy will look like the following:
+
+```json
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Effect": "Allow",
+            "Action": [
+                "s3:PutObject",
+                "s3:GetObject",
+                "s3:GetObjectVersion",
+                "s3:ListMultipartUploadParts",
+                "s3:AbortMultipartUpload",
+                "s3:ListBucket",
+                "s3:ListBucketVersions",
+                "s3:ListBucketMultipartUploads",
+                "s3:GetBucketOwnershipControls",
+                "s3:GetBucketPublicAccessBlock",
+                "s3:GetBucketObjectLockConfiguration",
+                "s3:GetBucketVersioning",
+                "s3:GetBucketLocation"
+            ],
+            "Resource": [
+                "arn:aws:s3:::teleport-longterm-<unique_id>",
+                "arn:aws:s3:::teleport-longterm-<unique_id>/events/*",
+                "arn:aws:s3:::teleport-longterm-<unique_id>/sessions/*",
+                "arn:aws:s3:::teleport-transient-<unique_id>",
+                "arn:aws:s3:::teleport-transient-<unique_id>/query_results/*"
+            ],
+            "Sid": "ReadWriteSessionsAndEvents"
+        },
+        {
+            "Effect": "Allow",
+            "Action": [
+                "athena:StartQueryExecution",
+                "athena:GetQueryResults",
+                "athena:GetQueryExecution"
+            ],
+            "Resource": "arn:aws:athena:us-west-2:278576220453:workgroup/teleport_events_<unique_id>",
+            "Sid": "AllowAthenaQuery"
+        },
+        {
+            "Effect": "Allow",
+            "Action": [
+                "glue:GetTable",
+                "glue:GetTableVersion",
+                "glue:GetTableVersions",
+                "glue:UpdateTable"
+            ],
+            "Resource": [
+                "arn:aws:glue:us-west-2:278576220453:catalog",
+                "arn:aws:glue:us-west-2:278576220453:database/teleport_events_<unique_id>",
+                "arn:aws:glue:us-west-2:278576220453:table/teleport_events_<unique_id>/teleport_events"
+            ],
+            "Sid": "FullAccessOnGlueTable"
+        }
+    ]
+}
+```
+
+## Regional storage
+
+Currently it is a requirement that the AWS infrastructure is hosted in the same
+AWS region as the Teleport Auth Service for your cluster.
+Historically this has always been us-west-2, but new clusters may be deployed to
+other regions in the future.
```

Comment:
Not sure if we want to "leak" this here

Reply:
It's fine. We already mention us-west-2 in a few other places.